hasura.planx.uk/migrations/1736953485265_populate published_flows has_send_component/down.sql

```diff
@@ -1,2 +1,12 @@
+WITH ranked_flows AS (
+  SELECT *,
+    ROW_NUMBER() OVER (PARTITION BY flow_id ORDER BY created_at DESC) as rn
+  FROM "public"."published_flows"
+)
 UPDATE "public"."published_flows"
 SET has_send_component = false
+WHERE id IN (
+  SELECT id 
+  FROM ranked_flows 
+  WHERE rn = 1
+);
```

Comment:
This looks like a hard way of writing `LIMIT 1`

Docs: https://www.postgresql.org/docs/current/queries-limit.html